scripts/release/changelog.js

```diff
@@ -9,10 +9,14 @@
 
 'use strict';
 
+// In order to use the same version for all packages (including builds and ckeditor5 itself), you can call:
+// npm run changelog [newVersion]
+
 const devEnv = require( '@ckeditor/ckeditor5-dev-env' );
 const commonOptions = {
 	cwd: process.cwd(),
-	packages: 'packages'
+	packages: 'packages',
+	newVersion: process.argv[ 2 ] || null
```

Comment:
New version of what? Does this option make sense in CKEditor 5?

Reply:
For the real release - nope. But for testing (like I often do) - yes.

We can skip that and I'll modify this file when I'll need that.

Reply:
newVersion for changelog of course. It's a version that you need to specify based on history of commits.

Reply:
> newVersion for changelog of course. It's a version that you need to specify based on history of commits.

For which changelog? This task builds 30x changelogs. Does it affect all pkgs?

Reply:
Yes.

Reply:
Anyway, I just want to have a clarity here. You're adding an option so you should be clear why. If it's an option for tests, then please add a comment clarifying this in the code.

Reply:
That was my goal. It speeds up testing this tool. I don't lose time for parsing commits from repositories and pasting the same version as an answer for the question "type new version".

Reply:
It could even be quite useful for me in the future, once I'd have to release for e.g. the next major version of every package because of something. Then, we could make this option to also accept values such as `major` or `next-alpha`. But that's for the future, of course.

Reply:
Reported as a new issue: https://github.com/ckeditor/ckeditor5-dev/issues/421